idp.go

```diff
@@ -109,6 +110,33 @@ func (idp *IdentityProvider) MetaDataResponse() (string, *Reject) {
 			})
 		}
 	}
+	
+	if idp.Organization != nil {
+		metadata.Organization = append(metadata.Organization, lib.Organization{
```

Comment:
@sccalabr metadata.Organization  is not array type, so instead of append, need to directly assign to the metadata.Organization